batch/test/test_batch.py

```diff
@@ -11,7 +11,9 @@
 
 class Test(unittest.TestCase):
     def setUp(self):
-        self.batch = batch.client.BatchClient(url=os.environ.get('BATCH_URL'))
+        self.batch = batch.client.BatchClient(
+            url=os.environ.get('BATCH_URL'),
+            headers={'User': 'dking@broadinstitute.org'})
```

Comment:
I don't think we should have your email in there.

Reply:
switched to `ci`